src/getCoinCombination.test.js

```diff
@@ -7,4 +7,39 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return only pennies if input is from 1 to 4', () => {
+    expect(getCoinCombination(2))
+      .toEqual([2, 0, 0, 0]);
+  });
+
+  it('should return only quarters if input divisibles by 25', () => {
+    expect(getCoinCombination(100))
+      .toEqual([0, 0, 0, 4]);
+  });
+
+  it('should return exact amount of each coin', () => {
```

Comment:
please, in the description add the number of the input - here it is 41, and your expected result - like, one dime, one nickel and so on should be returned.
So please, in the name of the test make a full summary, not only what is the expected result) Do it for all of your checks for readability)